policyengine_us/variables/gov/states/ny/nysdps/categories/ny_clean_heat_building_category.py

```diff
@@ -1,16 +1,16 @@
 from policyengine_us.model_api import *
 
 class BUILDCategory(Enum):
```

Comment:
It can be a boolean, but I made it an enumerated type to align with other conditions determining the incentive amount. Also, to avoid unnecessary misunderstanding, I'd like to use the exact wording from the policy. (below is the diagram showing the classification)
<img width="843" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/142937336/82e493bb-6d80-4665-b976-c32c89295ae6">


Do you think it should be a boolean? If we make this a boolean, it would be good also to make DAC (whether it's DAC or not) a boolean.

Reply:
sorry for delay - yes please make both bools. enums with two categories are ok if it could plausibly be expanded to more categories, but that doesn't seem the case here